test/conversion_helper-test.js

```diff
@@ -428,23 +428,19 @@ describe('conversion_helper', () => {
       let expectedOutput = [];
       if (Array.isArray(numbers)) {
         expectedOutput = numbers.reduce((memo, el) => {
-          let inputNumber;
-          let inputUnit;
-
-          if (Array.isArray(el)) {
-            inputNumber = el[0];
-            inputUnit = el[1];
-          } else {
-            inputNumber = el.toString();
-            inputUnit = unit;
-          }
+          let inputNumber = [];
+          el.forEach(function(item) {
+            inputNumber.push(item.toString());
+          });
+          let inputUnit = unit;
 
           memo.push(createImperialMap(inputNumber, inputUnit));
 
           return memo;
         }, expectedOutput);
       }
 
+      console.log(input);
```

Comment:
There are some extra console.log statements in the code! 

If you move the "pre-commit" file into your "./git/hooks" directory, it will stop you from committing code that 1) has excess console.log statements and 2) are failing tests.